core/src/OpenACC/Kokkos_OpenACC_Instance.hpp

```diff
@@ -12,6 +12,9 @@
 #include <iosfwd>
 #include <string>
 
+#define KOKKOS_IMPL_ACC_ASYNC_RANGE_BEGIN 64
+#define KOKKOS_IMPL_ACC_ASYNC_RANGE_LENGTH 128
```

Comment:
I would prefer if these were constexpr variable within the scope of `impl_partition_space`.

Reply:
Changed.